front/app/modules/commercial/flag_inappropriate_content/admin/components/EmptyMessageModerationsWithFlag.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+// i18n
+import { FormattedMessage } from 'utils/cl-intl';
+import messages from './messages';
+
+interface Props {}
```

Comment:
You can just remove this, instead of leaving it empty here and unused below.

Reply:
We keep it out of convention to indicate there are still props (just an empty object). We indicate non-used props with an underscore (like `_props` in this file). Likewise, if you think it's important to have different conventions, feel free to add it as an item to the CDM! :-)